keyboards/chidori/keymaps/oled_sample/keymap.c

```diff
@@ -0,0 +1,228 @@
+/* Copyright 2019 ENDO Katsuhiro <ka2hiro@kagizaraya.jp>
+ * Copyright 2020 Masaya Uno
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#include "board.h"
+#include <stdio.h>
+
+enum layer_number { _QWERTY = 0, _COLEMAK, _DVORAK, _LOWER, _RAISE, _ADJUST };
```

Comment:
From a readability standpoint, this may be easier to read: 
```suggestion
enum layer_number { 
    _QWERTY,
     _COLEMAK, 
     _DVORAK, 
     _LOWER, 
     _RAISE, 
     _ADJUST,
};
```

Also, no need to specify 0, as it starts there.